packages/packages/history/src/init.ts

```diff
@@ -0,0 +1,14 @@
+import { registerToggleAction } from '@elementor/top-bar';
+import useActionProps from './hooks/use-action-props';
+
+export default function init() {
+	registerTopBarMenuItems();
+}
+
+function registerTopBarMenuItems() {
+	registerToggleAction( 'main', {
+		name: 'open-history',
+		priority: 20,
```

Comment:
check out the priorities I used in other PRs to make sure it works properly

Reply:
As agreed, will keep it as it is.